### ISSUE TITLE
Carousel component navigation difficult to see in light theme

### ISSUE THREAD

**Issue by kellyhutchins** [Esri/calcite-design-system#11426]
### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Summary

The dots and navigation arrows are challenging to see in the light theme on both my Dell P2212H external monitor and built-in MacBook Pro display, making it difficult to navigate the carousel content.

### Actual Behavior

The dots and navigation arrows are non-text components so I think they would need to meet the color contrast requirements outlined [here](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 
The exceptions in the doc include inactive and disabled elements but I'd argue that the dots and navigation arrows are not inactive since they are meant to be interacted with by the user.

![Image](https://github.com/user-attachments/assets/8ab03015-11b3-4e47-af16-e42d4e8300d9)

To confirm that the issue was not isolated to me, I opened this [sample link](https://next.sites.afd.arcgis.com/javascript/latest/sample-code/legend-card/) and requested individuals to move to the subsequent slide in the carousel. Out of the three individuals I enlisted to assist with the test, two were unable to locate the navigation arrows or dots while using the Dell monitor. These elements are slightly more discernible when using the MacBook's built-in display. However, the testers presumed that there was only one slide available due to the faint visibility of the dots and arrows.

### Expected Behavior

I think the navigation controls (dots and arrows) should meet the color contrast requirements outlined in the Success Criteria doc. 

### Calcite Design Solution:

- [ ] Update the chevrons to use `color.text.3`
- [ ] Update the page dots to use `color.border.1`

![Image](https://github.com/user-attachments/assets/586b9fde-cbd5-4184-88b7-a7650191bf6b)

[Figma file](https://www.figma.com/design/vB6xFCAFeW0zKrT4nKlh8Z/branch/oelI3Ny7uB7KJ4SnhxALmV/Calcite-UI-Kit-%26-Styles?review-number=62&t=hnR533rUm2eQTCCf-0) for reference.

### Reproduction Sample

https://next.sites.afd.arcgis.com/javascript/latest/sample-code/legend-card/

### Reproduction Steps

1. Open the sample
2. Note that the navigation (arrows and dots) are difficult to see 
3. Inspect the values with the dev tools and note that they do not meet color contrast requirements for non text elements. 

### Reproduction Version

3

### Priority impact

impact - p3 - not time sensitive

### Calcite package

- [x] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Instant Apps

**Comment by ashetland** [comment]
@macandcheese @SkyeSeitz @geospatialem I added [Calcite Design Solution](https://github.com/Esri/calcite-design-system/issues/11426#:~:text=Calcite%20Design%20Solution%3A) above and marking as ready for dev.

**Comment by geospatialem** [comment]
Verified the contrast is met with the [previous/next chevron buttons](https://webaim.org/resources/contrastchecker/?fcolor=6A6A6A&bcolor=FFFFFF) and enhanced contrast is provided on the next/previous dots

![Image](https://github.com/user-attachments/assets/3b5b132c-9796-407e-a10c-01310e734420)
